docs/Analog.md

```diff
@@ -1,32 +1,83 @@
 
-## PCB Layout
 
-Discussion related to PCB layout for analog subsystem:
-- What area of board is it on?
-- Noise??
+_V_<sub>i</sub> = _V_<sub>REF</sub>  + 0.1 (_V_<sub>i</sub><sup>+</sup> - _V_<sub>i</sub><sup>-</sup>)
 
-## Max Voltage Limitations
+where i = 1, 2, 3 ..., 8. (_V_<sub>i</sub><sup>+</sup> - _V_<sub>i</sub><sup>-</sup>) is bipolar differential input coming from analog connectors and _V_<sub>i</sub> is output of the op amp which is fed to the low-pass filter (next subsection). _V_<sub>REF</sub> = 2.048 V is supplied from AMDC and 0.1 gain is set using laser trimmed resistors during IC fab, so each resistance value internal to the [INA2143UA](http://www.ti.com/lit/ds/symlink/ina143.pdf) is very accurate. This results in very high CMRR. Differential input to the op amp can be between ±20 V (AC/DC, each input between ±10 V), which means the output can be in the range of 0.048 V and 4.048 V.
+
+More detailed information on the operating conditions of the op amp can be found in the [datasheet](http://www.ti.com/lit/ds/symlink/ina143.pdf).
+ 
+### Low-Pass Filters (LPFs)
+After analog input voltage levels are decreased using op amps, the next stage is to remove high-frequency noises using low-pass filters. To implement this, 8 simple first-order RC filters are used each for 1 analog input. The cutoff frequency of the filter is selected based on the resistance and capacitance values:
+
+_f_<sub>c</sub> = 1 / (2 π _RC_)
+
+The main purpose of these LPFs, however, is for anti-aliasing. They are set to 50 kHz (resistor with _R_ = 100 Ω and capacitor with _C_ = 33 nF) since sampling is nominally done at 100 kHz for ADC, and control loop is nominally at 10 or 20 kHz.
+
+### ADC
+
+After analog input signal passes through analog front-end (voltage level decreased and high-frequeny noise removed), it can be inputted to ADC to convert to digital signal. [LTC2320-14](https://www.analog.com/media/en/technical-documentation/data-sheets/232014fa.pdf) high speed octal 14-bit + sign successive approximation register (SAR) ADC is used for this purpose which can receive up to 8 inputs. ADC chip is supplied by 5 V from AMDC to operate.
+
+Each analog input to ADC is configured as pseudo-differential bipolar signal - its positive input signal is single-ended LPF output that can swing between 0.048 V and 4.048 V, and its negative input signal is at _V_<sub>REF</sub> = 2.048 V, resulting in differential input span of ±2 V which is digitized by ADC.
+
+The outputs of the ADC chip are 8 serial data outputs each resulting from 1 differential analog input. The chip is also supplied by 1.8 V which is used as a digital HIGH of the output signal. These 8 digital signals are then transmitted to Picozed. There are other 3 digital signals (ADC CNV, ADC SCK and ADC CLKOUT) needed for the interface with FPGA.

```

Comment:
"transmitted to Picozed"... 

It would be nice to provide info here that a verilog developer using Vivado would find useful. Which Picozed pins? How should they be configured in Vivado?

Reply:
Looks like it is addressed by @npetersen2 in c4c05c9